keps/sig-cluster-lifecycle/kubeadm/20190506-kubeadm-machine-output.md

```diff
@@ -0,0 +1,370 @@
+---
+title: kubeadm-machine-output
+authors:
+  - "@akutz"
+  - "@bart0sh"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+reviewers:
+approvers:
```

Comment:
@timothysc 
@neolit123 
@fabriziopandini

Reply:
Are these people who are required for approval or will approve or have approved? I'm updating the reviewers list, but I wasn't sure if I should add approvers until something is approved.